libfprint: 1.90.5 -> 1.90.7

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/libfprint/versions.


meta.description for libfprint is: "A library designed to make it easy to add support for consumer fingerprint readers"

meta.homepage for libfprint is: "https://fprint.freedesktop.org/"

meta.changelog for libfprint is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 1.90.7 with grep in /nix/store/5a0fdsvpcbc49awsnhdswbaxlaasz74v-libfprint-1.90.7
- directory tree listing: https://gist.github.com/3b0a0bbe20ef6de1b3d95a71142d1cad

- du listing: https://gist.github.com/fe2d92a6b8b89a55017cd22b90c447ab

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
6 total rebuild path(s)

2 package rebuild(s)

2 x86_64-linux rebuild(s)
2 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
2 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
fprintd
libfprint
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/5a0fdsvpcbc49awsnhdswbaxlaasz74v-libfprint-1.90.7 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libfprint https://github.com/r-ryantm/nixpkgs/archive/5509313b7bc97f7a619953f201838df46ac01da5.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/5a0fdsvpcbc49awsnhdswbaxlaasz74v-libfprint-1.90.7
ls -la /nix/store/5a0fdsvpcbc49awsnhdswbaxlaasz74v-libfprint-1.90.7/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>fprintd</li>
    <li>libfprint</li>
  </ul>
</details>

---

###### Maintainer pings

cc @abbradar for testing.

cole-h commented:
@ofborg eval

jtojnar reviewed (approved):
Changes look okay:

- https://gitlab.freedesktop.org/libfprint/libfprint/-/tags/v1.90.7
- https://gitlab.freedesktop.org/libfprint/libfprint/-/tags/v1.90.6

jtojnar committed:
libfprint: 1.90.5 -> 1.90.7

- https://gitlab.freedesktop.org/libfprint/libfprint/-/tags/v1.90.7
- https://gitlab.freedesktop.org/libfprint/libfprint/-/tags/v1.90.6

Also correct license, drop no longer needed cflags and avoid alias.

jtojnar committed:
fprintd: 1.90.1 → 1.90.9

https://gitlab.freedesktop.org/libfprint/fprintd/-/blob/v1.90.9/NEWS#L1-73

jtojnar committed:
libfprintd: enable tests

virtual-image driver is needed for fprintd tests and enabling it
required patching shebangs of test-related scripts so I have enabled
tests here as well.

jtojnar commented:
Also looked at https://gitlab.freedesktop.org/libfprint/fprintd/-/blob/v1.90.9/NEWS#L1-73 and updated it as well. cc @Elyhaka

tomberek commented:
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110034` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>fprintd</li>
<li>libfprint</li>
</ul>
</details>
